homeassistant/components/synology_dsm/binary_sensor.py

```diff
@@ -97,7 +97,7 @@ class SynoDSMUpgradeBinarySensor(SynologyDSMBaseEntity, BinarySensorEntity):
     @property
     def is_on(self) -> bool:
         """Return the state."""
-        return getattr(self._api.upgrade, self.entity_type)
+        return getattr(self._api.upgrade, self.entity_type) and True
```

Comment:
Not sure what's the purpose of this. Maybe wrap it with `bool()` instead?

Reply:
yeah you are right, I already done it in line 72, but totally oversee this both :see_no_evil: